### PR TITLE
Fix #210 - Update readme with the latest spec details

### LINKED ISSUE #210: Update MicroProfile IO website
Looks like the MicroProfile website is not up to date.
https://microprofile.io/project/eclipse/microprofile-health

It still mentions for instance @Health which is now deprecated.

### PR DESCRIPTION
**Pull request by rafabene**

### CODE REVIEW

**Comment by rafabene** [comment]
@xstefank @antoinesd @johnament @cescoffier Can you take a look at this PR, please? Thanks

File: README.adoc [hunk 1]
```diff
@@ -85,47 +85,78 @@ public class SuccessfulCheck implements HealthCheck {
...
+Any enabled bean with a bean of type `org.eclipse.microprofile.health.HealthCheck` and `@Liveness`, `@Readiness` or `@Health` qualifier can be used as health check procedure.
+
+
+Contextual references of health check procedures are invoked by runtime when the outermost protocol entry point (i.e. `http://HOST:PORT/health`) receives an inbound request
+
+
+```
 @ApplicationScoped
-public class CheckDiskSpace implements HealthCheck {
+public class MyCheck implements HealthCheck {
```
Comment (xstefank) [inline README.adoc:132]: You are missing a qualifier here.

Comment (rafabene) [reply README.adoc:134]: Fixed at Readme and java-api adocs.

**Comment by xstefank** [comment]
Thanks @rafabene

**Comment by rafabene** [comment]
@xstefank Fixed. Thanks

**Pull request merged**